Tools/ardupilotwaf/boards.py

```diff
@@ -48,6 +48,10 @@ def srcpath(path):
         env.SRCROOT = srcpath('')
         self.configure_env(cfg, env)
 
+        env.DEFINES.update(
+            ENABLE_SCRIPTING = 0,
```

Comment:
this would break lots of code, as it uses "#ifdef" now

Reply:
Patches have been added for that.